Demo/src/androidTest/java/com/braintreepayments/demo/test/DropInTest.java

```diff
@@ -121,6 +121,9 @@ public void performsThreeDSecure2FrictionlessVerification() {
         onDevice(withTextStartingWith("created")).check(text(endsWith("authorized")));
     }
 
+    @Ignore("Test fails due to Cardinal Activity changes that resulted in the authorization code " +
```

Comment:
Nitpick: I think these tests are unrelated to this PR. We should commit this on master first and then rebase this PR.